tests/switchless/host/host.c

```diff
@@ -151,26 +160,47 @@ int main(int argc, const char* argv[])
              &enclave)) != OE_OK)
         oe_put_err("oe_create_enclave(): result=%u", result);
 
-    char out[STRING_LEN];
-    int return_val;
-
+    // Measure switchless ocall performance.
     uint64_t num_extra_enc_threads = num_enclave_threads - 1;
-    oe_thread_t tid[32] = {0};
+    thread_info_t tinfo[32];
```

Comment:
Do we check `num_extra_enc_threads` against `32` anywhere?

Reply:
I have defined NUM_TCS=32 in switchless.edl and used that in both host and enclave.